sql/hypertable_constraint.sql

```diff
@@ -38,4 +38,4 @@ BEGIN
     END IF;
```

Comment:
Also in line 33 of this file, the `search_path` is adjusted. Adding `pg_temp` seems not to be necessary at the moment since no relations are looked up in the function. However, this might change in the future and the developer might forget to adjust the `search_path`.

Reply:
good catch